angel.py

```
@@ -1,9 +1,10 @@
 # @SudoR2spr WOODcraft 
 from asyncio import events
+
 import telebot
 from telebot import types
 import requests
-from bs4 import BeautifulSoup
+from bs4 import BeautifulSo
```

Ruff (0.8.2)

`bs4.BeautifulSo` imported but unused

Remove unused import: `bs4.BeautifulSo`

(F401)